modules/sbo-rbac-requirements.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *openshift-docs/applications/connecting_applications_to_services/exposing-binding-data-from-a-service
+[id="sbo-rbac-requirements_{context}"]
+= RBAC requirements
+
+To expose the backing service binding data using the {servicebinding-title}, you require certain Role-based access control (RBAC) permissions. You must specify certain verbs under the `rules` field of the `ClusterRole` resource to allow the RBAC persmissions for the backing service resources. Defining these `rules` enables the {servicebinding-title} with permissions to read the binding data of the the backing service resources throughout the cluster.
```

Comment:
```suggestion
To expose the backing service binding data using the {servicebinding-title}, you require certain Role-based access control (RBAC) permissions. Specify certain verbs under the `rules` field of the `ClusterRole` resource to allow the RBAC permissions for the backing service resources. When you define these `rules`, you permit/allow the {servicebinding-title} to read the binding data of the the backing service resources throughout the cluster.
```
The last sentence could be tricky grammatically for some users or localization teams. Adding "you" could simplify it a bit but isn't necessary.

Reply:
@missmesss Good catch! Thanks, this definitely helps :)